.run/save-frontend_run --continuous.run.xml

```diff
@@ -1,6 +1,11 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="save-frontend:run --continuous" type="GradleRunConfiguration" factoryName="Gradle">
     <ExternalSystemSettings>
+      <option name="env">
+        <map>
+          <entry key="NODE_OPTIONS" value="--openssl-legacy-provider" />
```

Comment:
export NODE_OPTIONS=--openssl-legacy-provider

Reply:
fuckers have broken community again: https://github.com/webpack/webpack/issues/14532

Reply:
Looks like need to enable it somehow in CI/CD build also

Reply:
> Looks like need to enable it somehow in CI/CD build also

It can be added using `env:` key in gradle step in gh workflow

Reply:
Not working with env due to a new issue: https://gitanswer.com/vscode-running-code-via-cli-triggers-error-openssl-legacy-provider-is-not-allowed-in-node-options-typescript-1046666817